pycyphal/transport/udp/_ip/_link_layer.py

```diff
@@ -17,7 +17,6 @@
 import pycyphal
 from pycyphal.transport import Timestamp
 
-
 _logger = logging.getLogger(__name__)
```

Comment:
(╯°□°)╯︵┻━┻

Reply:
┬━┬ ノ( ゜-゜ノ)

This is part of the black code style I guess? 😄

Reply:
No. It allows either one or two blank lines between top-level statements, so if you accidentally remove one here, it will let you keep it. Which is weird, if you ask me.